earthaccess/store.py

```diff
@@ -612,11 +615,11 @@ def _download_file(self, url: str, directory: Path) -> str:
                         # https://docs.python-requests.org/en/latest/user/quickstart/#raw-response-content
                         shutil.copyfileobj(r.raw, f, length=1024 * 1024)
             except Exception:
-                print(f"Error while downloading the file {local_filename}")
-                print(traceback.format_exc())
+                logger.error(f"Error while downloading the file {local_filename}")
+                logger.error(traceback.format_exc())
```

Comment:
Using `logger.exception` will include the traceback, so there's no need for an additional call.

```suggestion
                logger.exception(f"Error while downloading the file {local_filename}")
```